admin-dev/themes/default/js/tree.js

```diff
@@ -214,23 +215,24 @@ Tree.prototype = {
           useCheckBox,
         },
         (content) => {
-          $(`#${idTree}`).html(content);
-          organizeTree();
-          $(`#${idTree}`).tree('init');
-          that.$element.find('label.tree-toggler').each(
+          targetTree.html(content);
+          // Function organizeTree() disabled because it no longer has known usage - Crezzur
+          // organizeTree();
```

Comment:
Remove it if it's unused

Reply:
Ping @NeOMakinG

Reply:
The call have been deleted too quick. It may introduce regressions somewhere else. We are talking about this and I already did a PR to restore it
https://github.com/PrestaShop/PrestaShop/pull/24312

Reply:
This merge PR is blocked utnil we know what to do

Reply:
No regression found 🌈